modules/nextflow/src/main/groovy/nextflow/processor/TaskConfig.groovy

```diff
@@ -412,6 +412,10 @@ class TaskConfig extends LazyMap implements Cloneable {
         new PodOptions((List)get('pod'))
     }
 
+    String getComputeResourceType() {
+        get('computeResourceType') as String
+    }
```

Comment:
Is this really necessary? I'd avoid to introduce a process directive specific for this

Reply:
so you want to move it from process to pod options?

Reply:
think, it makes more sense